### ISSUE TITLE
All play Amazon Music Playlist

### ISSUE THREAD

**Issue by Apollon77** [Apollon77/alexa-remote#3]
command = "{\"playlistId\":\"" + playListId + "\",\"playQueuePrime\":true}";
"POST",
                    alexaServer + "/api/cloudplayer/queue-and-play?deviceSerialNumber=" + device.serialNumber
                            + "&deviceType=" + device.deviceType + "&mediaOwnerCustomerId="
                            + device.deviceOwnerCustomerId + "&shuffle=false",
command

**Comment by skillix** [comment]
hi, weren't there any changes concerning the playlist, radio and music track?

`Sending Request with {"host":"alexa.amazon.fr","path":"/api/cloudplayer/queue-and-play?deviceSerialNumber=G090U61090960J55&deviceType=A32DOYMUN6DTXA&mediaOwnerCustomerId=A3T9PVE5NO04DQ&shuffle=false&_=1629646989233","method":"POST","timeout":10000,"headers":{}} and data={"playlistId":"d2993f4f-2bad-435f-92e7-983915a6d69f","playQueuePrime":true}
Responce: {"error":null}`

**Comment by Apollon77** [comment]
I do not understand your question

**Comment by skillix** [comment]
for about a week no user can play a playlist, radio or music track. There is an error with no information, and it always plays the last media played. Does it still work for you or amazon to change a little thing?

**Comment by Apollon77** [comment]
Radio stations using tunein direcly seems to be an issue on amazon side ... I do not know more errors until now ... but if it is "since a week" then it is most likely amazon issue

**Comment by skillix** [comment]
Yes I suspected, but I wanted to confirm that you also had the problem to know if the problem came from amazon or from our plugin.

Do you also have the problem with the playlists?

**Comment by Apollon77** [comment]
Honestly: Me personally not (also not because I'm on vacation in denmark since a week ;-))) ) ... so "unconfirmed" from my side

**Comment by skillix** [comment]
good it smells bad I reproduced the problem on https://alexa.amazon.fr/spa/index.html#player and I saw this message "It is no longer possible to browse the music of Amazon Music on browsers. To browse music from Amazon Music, download the latest version of the Alexa mobile app from the Appstore of your mobile operating system and go to the Play tab. " on an annex page
